docs/evaluation/how_to_guides/datasets/manage_datasets_in_application.mdx

```diff
@@ -76,6 +76,10 @@ the run before adding it to the dataset.
 
```

Comment:
The wording in the lines above (can't select since I didn't edit) doesn't make sense to me. I think we should change? @agola11